x/tier/types/events.go

```diff
@@ -0,0 +1,5 @@
+package types
+
+const (
+	EventTypeCancelUnlocking = "cancel_unlocking"
```

Comment:
suggestion: Nothing todo with this exact line, just commenting here since it provides context.

We should probably map all the various/necessary/potential event types that would be of value for the `tier` module and enumerate them all here.